include/ttmlir/Dialect/TTKernel/IR/TTKernelOps.td

```diff
@@ -65,20 +65,6 @@ def TTKernel_UnpackABOp : TTKernel_Op<"unpack_ab"> {
     let arguments = (ins TTKernel_CB:$cb_a, I32:$src_a_index, TTKernel_CB:$cb_b, I32:$src_b_index);
 }
 
-def TTKernel_AcquireDstOp : TTKernel_Op<"acquire_dst"> {
-    let summary = "Aquire dest call.";
-    let description = [{
-      Aquire dest operation
-    }];
-}
-
-def TTKernel_ReleaseDstOp : TTKernel_Op<"release_dst"> {
-    let summary = "Release dest call.";
-    let description = [{
-      Release dest operation
-    }];
-}
-
```

Comment:
Thank you!  I think this might break `test/python/simple_kernel.py`, if possible we can update that test to use tile regs acquire/release instead.

Reply:
> What I also dislike is that we copied over op descriptions from metal comments

I agree, we can update all of the uses of `acquire_dst` with `tile_regs_acquire` and same for release etc.

> Let me know if I should leave these ops still or remove them.

Let's remove them! As long as we're green in CI then we're good.